### PR TITLE
:pager: Only deploy when timestamps aren't the same

### PR DESCRIPTION
**Pull request by WyriHaximus**

### CODE REVIEW

File: .github/workflows/deployment.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ name: Deployment
 # events but only for the master branch
 on:
   schedule:
-    - cron: "0 */1 * * *"
+    - cron: "13 * * * *"
```
Comment (thefrontendwizard) [inline .github/workflows/deployment.yml:9]: well, this effectively makes this action run only once a day, is that right?

File: .github/workflows/deployment.yml [hunk 2]
```diff
@@ -16,5 +16,12 @@ jobs:
...
     steps:
+      - name: Data Source timestamp
+        id: datasource
+        run: printf "::set-output name=timestamp::%s" $(curl -s https://brasil.io/api/dataset/covid19 | jq -r '.tables[1].import_date')
+      - name: Deployed timestamp
+        id: deployed
+        run: printf "::set-output name=timestamp::%s" $(curl -s https://mapadocovid19.com.br/api/last-updated | jq -r '.lastUpdate')
       - name: Deploy to Zeit
         run: curl -X POST https://api.zeit.co/v1/integrations/deploy/QmaoEk2yPVgwrT5ck9pGwiUXuBqjm8nVFeenDA6cqdFNZL/omrxg48jkR
+        if: steps.release.datasource.timestamp != steps.release.deployed.timestamp
```
Comment (thefrontendwizard) [inline .github/workflows/deployment.yml:27]: ❤️ love how pretty this looks, thank you very much for it

Comment (WyriHaximus) [reply .github/workflows/deployment.yml:27]: Wanted to go for a one liner, but this also makes it very simple to understand what's going on (and I didn't have to deal with bash one liner if statements :P.).

File: .github/workflows/deployment.yml (hunk 1 above)
Comment (WyriHaximus) [reply .github/workflows/deployment.yml:9]: It runs it every hour at minute 13: https://crontab.guru/#13_*_*_*_*

Comment (thefrontendwizard) [reply .github/workflows/deployment.yml:9]: should we do it more since it'll only deploy if it changes?

Comment (WyriHaximus) [reply .github/workflows/deployment.yml:9]: We could, but lets see how often is deploys first. Every 5 minutes should give you the most up to data data, but I want to add something else before that first ;)

**Pull request merged**